Pending_updates/com_007_FDI_regulatory_restrictiveness_index/com_007_FDI_regulatory_restrictiveness_index_processing.py

```diff
@@ -0,0 +1,102 @@
+import pandas as pd
+import os
+import sys
+utils_path = os.path.join(os.path.abspath(os.getenv('PROCESSING_DIR')),'utils')
+if utils_path not in sys.path:
+    sys.path.append(utils_path)
+import util_files
+import util_cloud
+import util_carto
+from zipfile import ZipFile
+import shutil
+import logging
+import glob
+
+# Set up logging
+# Get the top-level logger object
+logger = logging.getLogger()
+for handler in logger.handlers: logger.removeHandler(handler)
+logger.setLevel(logging.INFO)
+# make it print to the console.
+console = logging.StreamHandler()
+logger.addHandler(console)
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+# name of table on Carto where you want to upload data
+# this should be a table name that is not currently in use
+dataset_name = 'com_007_FDI_regulatory_restrictiveness_index' #check
+
+logger.info('Executing script for dataset: ' + dataset_name)
+# create a new sub-directory within your specified dir called 'data'
+# within this directory, create files to store raw and processed data
+data_dir = util_files.prep_dirs(dataset_name)
+
+'''
+Download data and save to your data directory
+Data can be downloaded at the following link:
+http://stats.oecd.org/Index.aspx?datasetcode=FDIINDEX#
+Above the table, there is a 'export' button that will lead to a dropdown menu containing different export options.
+Once you select 'Text file (CSV)' from the menu, a new window will occur and allow you to download the data as a csv file to your Downloads folder.
+'''
+logger.info('Downloading raw data')
+download = glob.glob(os.path.join(os.path.expanduser("~"), 'Downloads', 'FDIINDEX_*.csv'))[0]
+
+# Move this file into your data directory
+raw_data_file = os.path.join(data_dir, os.path.basename(download))
+shutil.move(download,raw_data_file)
+
+# read in data to pandas dataframe
+df = pd.read_csv(raw_data_file)
+
+'''
+Process data
+'''
+# remove duplicate column 
+df = df.drop(columns = 'TIME')
+
+# remove columns with no values 
+df = df.drop(columns = ['Flag Codes', 'Flags'])
+
+# remove columns with only one unique value 
+df = df.drop(columns = ['Type of restriction', 'SERIES', 'Series', 'RESTYPE'])
+
+# remove the column 'SECTOR' since it contains the same information as the column 'Sector/Industry'
+df = df.drop(columns = 'SECTOR')
+
+# reformat the dataframe so each sector becomes a new column 
+pivoted = pd.pivot_table(df, index = ['LOCATION', 'Country','Year'], columns = 'Sector / Industry', values = 'Value').reset_index()
+
```

Comment:
```suggestion
# reformat the dataframe so each sector becomes a new column 
pivoted = pd.pivot_table(df, index = ['LOCATION', 'Country','Year'], columns = 'Sector / Industry', values = 'Value').reset_index()

# rename columns, replacing or removing symbols and
# making them all lowercase so that it matches Carto column name requirements
pivoted.columns = [re.sub('[().]', '', col.lower().replace('&', 'and')) for col in pivoted.columns]

```